tools/st2-setup-samples.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+PACK=$1
+if [[ "$PACK" == "examples" ]]; then
+    PATH="/usr/share/doc/st2/"
+elif [[ "$PACK" == "tests" ]]; then
+    PATH="/usr/share/stackstorm/"
+fi
```

Comment:
Would be good to add else and throw / exit if uses passes in an invalid / unsupported pack name.

```bash
...
else:
  echo "Invalid pack: ${PACK}
  exit 2
fi
...
```